frontend/src/component/FeedItem/Preview.jsx

```diff
@@ -270,19 +302,23 @@ export class Preview extends React.Component {
         )}
         ref={(ref) => this.receivePreviewRef(ref)}
       >
-        <Link to={PAGE.WORKSPACE.CONTENT(content.workspaceId, content.type, content.id)}>
-          {this.getPreviewComponent()}
-          {state.previewOverflow && (
-            <div className='feedItem__preview__overflowOverlay' />
-          )}
-        </Link>
+        {this.getPreviewWithLink()}
       </div>
     )
   }
 }
 
 Preview.propTypes = {
-  content: PropTypes.object.isRequired
+  fallbackToAttachedFile: PropTypes.boolean,
+  content: PropTypes.object.isRequired,
+  link: PropTypes.string.isRequired,
+  linkType: PropTypes.oneOf(Object.values(LINK_TYPE))
+}
+
+Preview.defaultProps = {
+  fallbackToAttachedFile: false,
+  linkType: LINK_TYPE.OPEN_IN_APP
 }
 
 export default translate()(Preview)
+export { LINK_TYPE }
```

Comment:
Normal?

Reply:
Maybe not, but this is not coming from the merge.